HealthTrack/app/src/androidTest/java/com/example/healthtrack/ExampleInstrumentedTest.java

```diff
@@ -17,6 +29,27 @@
  */
 @RunWith(AndroidJUnit4.class)
 public class ExampleInstrumentedTest {
+
+    @Rule
+    public ActivityTestRule<ExerciseActivity> activityTestRule = new ActivityTestRule<>(ExerciseActivity.class);
+
+    @Test
+    public void addToClickCounter() {
+        onView(withId(R.id.text_counter))

```

Comment:
This is to test the text fields. It's not all that great for testing the number of clicks the counter goes through.